shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -238,13 +239,48 @@ private void printShortWorkerInfo(List<WorkerInfo> workerInfoList) {
         int usedPercentage = (int) (100L * usedBytes / capacityBytes);
         usedPercentageInfo = String.format(" (%s%%)", usedPercentage);
       }
-      print(String.format(SHORT_INFO_FORMAT, info.getAddress().getHost(),
+      print(String.format(shortInfoFormat, info.getAddress().getHost(),
           info.getLastContactSec(), "capacity", FormatUtils.getSizeFromBytes(capacityBytes)));
-      print(String.format(SHORT_INFO_FORMAT, "", "", "used",
+      print(String.format(shortInfoFormat, "", "", "used",
           FormatUtils.getSizeFromBytes(usedBytes) + usedPercentageInfo));
     }
   }
 
+  /**
+   * Gets the info format according to the longest worker name.
+   * @param workerInfoList the worker info list to get info from
+   * @param isShort whether exists only one tier
+   * @return the info format for printing long/short work info
+   */
+  private String getInfoFormat(List<WorkerInfo> workerInfoList, boolean isShort) {
+    // order by worker name length desc
+    Collections.sort(workerInfoList, new Comparator<WorkerInfo>(){
+      @Override
+      public int compare(WorkerInfo worker1, WorkerInfo worker2) {
+        return len(worker2) - len(worker1);
+      }
+    });
+    int maxWokerNameLength = len(workerInfoList.get(0));
+    int firstIndent = 16;
+    if (firstIndent <= maxWokerNameLength) {
+      // extend first indent according to the longest worker name
+      firstIndent = Double.valueOf(Math.ceil(maxWokerNameLength * 1.2)).intValue();
```

Comment:
can we keep the horizontal gaps between each column the same?

Reply:
If you mean formatting like '%-16s%-16s%-16s%s', I think it is unnecessary since other columns will extend simultaneously.